### PR TITLE
Run code formatter on Mix.Dep.Converger

### PR DESCRIPTION
**Pull request by pgrzesik**

### CODE REVIEW

File: lib/mix/lib/mix/dep/converger.ex [hunk 1]
```diff
@@ -217,27 +222,37 @@ defmodule Mix.Dep.Converger do
...
+      Enum.map_reduce(list, false, fn other, match ->
         %Mix.Dep{app: other_app, opts: other_opts} = other
+
         if other_app == app and other.top_level and dep.top_level do
-          Mix.shell.error "warning: the dependency #{inspect dep.app} is " <>
-                          "duplicated at the top level, please remove one " <>
-                          "of them"
+          Mix.shell().error(
+            "warning: the dependency #{inspect(dep.app)} is " <>
+              "duplicated at the top level, please remove one " <> "of them"
```
Comment (josevalim) [inline lib/mix/lib/mix/dep/converger.ex]: Unnecessary concatenation here at the end, please remove it.

File: lib/mix/lib/mix/dep/converger.ex [hunk 2]
```diff
@@ -217,27 +222,37 @@ defmodule Mix.Dep.Converger do
...
             {other, match}
+
           in_upper? && other_opts[:override] ->
             {other |> with_matching_only(dep, in_upper?), true}
+
           converge?(other, dep) ->
-            {other |> with_matching_only(dep, in_upper?)
-                   |> with_matching_req(dep) |> merge_manager(dep, in_upper?), true}
+            {
+              other
```
Comment (josevalim) [inline lib/mix/lib/mix/dep/converger.ex]: Please remove the expression out of the tuple and assign it to a variable.

**Comment by josevalim** [comment]
Hi @pgrzesik! This PR has revealed some bugs in the formatter, so I will close it and tackle it personally. Great find and thank you!

**Comment by josevalim** [comment]
Fixed in master after formatter updates, thanks!

**Comment by pgrzesik** [comment]
Hello @josevalim ! Many thanks to you for you work on Elixir!

**Comment by josevalim** [comment]
You are welcome! :+1:

**Pull request closed without merging**